docs/source/release_notes/release-notes.rst

```diff
@@ -44,8 +44,14 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed|
+         - KVS migrations will no longer verify equality between the from and to KVSes for the sweep priority and progress tables.
+           Note that these tables are still *migrated* across, as they provide heuristics for timely sweeping of tables.
+           However, these tables are expected to change as the migration process performs writes to the to-KVS.
```

Comment:
I think the `from-kvs` is being swept and hence `sweep.priority` there is being updated. Theoretically, there should be nothing to sweep in the `to-kvs` as there is only the latest version of the data and migrations are offline.